lib/rubygems/server.rb

```diff
@@ -657,7 +657,7 @@ def root(req, res)
       "only_one_executable" => true,
       "full_name" => "rubygems-#{Gem::VERSION}",
       "has_deps" => false,
-      "homepage" => "http://docs.rubygems.org/",
+      "homepage" => "http://guides.rubygems.org/",
```

Comment:
@segiddins guides.rubygems.org does not offer a legitimate HTTPS certificate because it's hosted on github pages.  In order to solve for this, you'd likely want to setup something like CloudFlare to front end a legitimate cert for the domain.  If you'd still prefer HTTPS, I can switch it here, but I think that would confusing to users.